### ISSUE TITLE
Subscript #1 of the array ID has value 36 which is greater than the upper bound of 25

### ISSUE THREAD

**Issue by DusanJovic-NOAA** [NOAA-EMC/UPP#102]
```
/scratch2/NCEPDEV/fv3-cam/Dusan.Jovic/sufs/simple-ufs-aero/src/post/sorc/ncep_post.fd/MDL2P.f(2095): error #5560: Subscript #1 of the array ID has value 36 which is greater than the upper bound of 25
               ID(36)=2                                                          
---------------^                                                                 
compilation aborted for /scratch2/NCEPDEV/fv3-cam/Dusan.Jovic/sufs/simple-ufs-aero/src/post/sorc/ncep_post.fd/MDL2P.f (code 1)
make[2]: *** [sorc/ncep_post.fd/CMakeFiles/nceppost.dir/MDL2P.f.o] Error 1       
make[2]: *** Waiting for unfinished jobs....
``` 
This should be fixed:
```
diff --git a/sorc/ncep_post.fd/RQSTFLD.F b/sorc/ncep_post.fd/RQSTFLD.F
index 50410d1..313c509 100644
--- a/sorc/ncep_post.fd/RQSTFLD.F
+++ b/sorc/ncep_post.fd/RQSTFLD.F
@@ -41,7 +41,7 @@
                  IQ(MXFLD),IS(MXFLD),ISMSTG(MXFLD),               &
                  ISMFUL(MXFLD),ISMOUT(MXFLD),LVLS(MXLVL,MXFLD),   &
                  IDENT(MXFLD),IFILV(MXFLD),IAVBLFLD(MXFLD),       &
-                 ID(25),IGDS(18)
+                 ID(36),IGDS(18)
       real    :: DEC(MXFLD)
       integer :: num_post_afld
       integer,allocatable :: LVLSXML(:,:)
```

**Comment by fossell** [comment]
Alternative "fix" can be found in DTC_post branch provided by developer: commit 912393b1645.

**Comment by fossell** [comment]
@DusanJovic-NOAA - I realize because of the differing/conflicting release timelines of ufs public release and DTC_post, there may be bug fixes in DTC_post that might need to go into public-v8.  I'm not sure how to do identify which, but if you encounter more in debug mode, peruse the [DTC_post commits](https://github.com/NOAA-EMC/EMC_post/commits/DTC_post) and perhaps we already have a fix.  Another option, we could try to merge DTC_post into release/public-v8 and test; there are changes in structure outside of source code, but source code itself should be somewhat similar with some bug fixes.

**Comment by DusanJovic-NOAA** [comment]
Great. If ID(36) is not needed can you apply fix in 912393b to release/public-v8.

**Comment by DusanJovic-NOAA** [comment]
I think we are so close to the public release that I think is too risky to merge entire DTC_post branch into release/public-v8.

**Comment by fossell** [comment]
@DusanJovic-NOAA - Agreed, it's on our list for post-release.

**Comment by fossell** [comment]
Fixed with #103. Closing.
